permission.go

```diff
@@ -47,12 +47,12 @@ func (client *DefaultClient) permissionAllowed(grantedPermissions []Permission,
 }
 
 func (client *DefaultClient) applyUserPermissionResourceValues(
-	grantedPermissions []Permission, claims *JWTClaims) []Permission {
+	grantedPermissions []Permission, claims *JWTClaims, namespace string) []Permission {
```

Comment:
by adding `namespace` here, is this going to introduce breaking changes to iam-go-sdk consumer?

Reply:
I dont think so, because this private function so the client not be able to access it